source/includes/_claim_fund_api.md

```diff
@@ -0,0 +1,1126 @@
+| 404 Not Found | 204 | Partner Trx ID is not found | Indicates that the partner\_trx\_id parameter contains a value that either does not exist in the database or is associated with a product named CLAIM\_FUND. |
+| 400 Bad Request | 990 | Send Callback must be either true or false | Indicates that the send\_callback parameter contains a non-boolean value. |
+| 429 Too Many Requests | 429 | Too Many Requests | Indicates that the Client has sent too many requests within a given period, exceeding the allowed rate limit. |
+| 504 Gateway Timeout | 504 | Request Timeout | Indicates that the server does not receive a timely response from an OY\! Service. |
+| 500 Server Error | 999 | Oops\! Something went wrong\! Sorry for the inconvenience. \\n The application has encountered an unknown error. \\n We have been automatically notified and will be looking into this with the utmost urgency. | Indicates failures due to an unexpected issue on the server side, including unhandled NPEs and database issues. |
+
+## Transaction Callback
+
+> Response for waiting approval callback:
+
+```json
+{
+  "tx_status": "WAITING_APPROVAL",
+  "tx_status_description": "The Claim Fund transaction is awaiting approval",
+  "amount": 500000,
+  "partner_trx_id": "TRX-20250408-001",
+  "recipient_email": "recipient@mail.com",
+  "recipient_name": "Recipient",
+  "note": "note",
+  "claim_fund_link": "https://claimfund.oyindonesia.com/claim-fund/claim/251280aa-95cb-41f2-84c3-8d2b247aa2b4?token=1ac006ce-82d9-4cb6-9afd-a621ca8dafa0",
+  "expiration": "2025-04-08 12:10:00",
+  "created": "2025-04-08 02:10:00",
+  "updated": "2025-04-08 02:15:00"
+}
+```
+
+> Response for successful callback:
+
+```json
+{
+  "tx_status": "SUCCESS",
+  "tx_status_description": "The Claim Fund transaction has completed",
+  "amount": 500000,
+  "partner_trx_id": "TRX-20250408-001",
+  "recipient_email": "recipient@mail.com",
+  "recipient_name": "Recipient",
+  "note": "note",
+  "claim_fund_link": "https://claimfund.oyindonesia.com/claim-fund/claim/251280aa-95cb-41f2-84c3-8d2b247aa2b4?token=1ac006ce-82d9-4cb6-9afd-a621ca8dafa0",
+  "expiration": "2025-04-08 12:10:00",
+  "created": "2025-04-08 02:10:00",
+  "updated": "2025-04-08 02:15:00"
+}
+```
+
+> Response for failed callback:
+
+```json
+{
+  "tx_status": "FAILED",
+  "tx_status_description": "The Claim Fund transaction has failed",
+  "amount": 500000,
+  "partner_trx_id": "TRX-20250408-001",
+  "recipient_email": "recipient@mail.com",
+  "recipient_name": "Recipient",
+  "note": "note",
+  "claim_fund_link": "https://claimfund.oyindonesia.com/claim-fund/claim/251280aa-95cb-41f2-84c3-8d2b247aa2b4?token=1ac006ce-82d9-4cb6-9afd-a621ca8dafa0",
+  "expiration": "2025-04-08 12:10:00",
+  "created": "2025-04-08 02:10:00",
+  "updated": "2025-04-08 02:15:00"
+}
+```
+
+Our system will send a callback to your system when the claim fund transaction status changed to: WAITING_APPROVAL, SUCCESS, and FAILED.
```

Comment:
revisi: `Our system will send a callback to your system when the Claim Fund API transaction status changes to: WAITING_APPROVAL, SUCCESS, and FAILED.` @guntoroyk